stock_picking_import_serial_number/tests/common.py

```diff
@@ -17,8 +17,7 @@ def assertUniqueIn(self, element_list):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
-        cls.lot_obj = cls.env["stock.production.lot"]
+        cls.lot_obj = cls.env["stock.lot"]
```

Comment:
Here above you can simplify writing by:

`class CommonStockPickingImportSerial:`